### DISCUSSION TITLE
Can you double the number of patches to 200?

### DISCUSSION THREAD

**Discussion by Jpro600k** [image-et-son/p600fw#139] [Q&A] [answered]
Can you double the number of patches to 200?

**Comment by image-et-son** [comment, marked as answer]
Hi, basically impossible, unfortunately. Storage is done per page and there are not enough pages left so I would need to fit more than one patch into one page. But patches currently use more than half a page, so I'd be messy. Anyway, lots of code relies on "one page per patch".
